### ISSUE TITLE
Wall CompoundStructureLayersLocation returns list of nulls in R2023

### ISSUE THREAD

**Issue by rvida5** [albandechasteigner/GeniusLociForDynamo#27]
To start, thank you for the awesome package!

I've used your "Wall CompoundStructureLayersLocation" node successfully in the past, but I am now getting an issue in Dynamo Revit 2.16.1.6510. I was initially using version 2022.7.5 of the package, but then updated to the latest version 2023.7.13 and in both cases I get the same null result. Are you aware of any issues with this node in the most recent versions of dynamo? I feel like I've used it successfully in Revit 2023 before but this may be the first time I'm running it in 2023.1.

Thank you for your time!

![image](https://github.com/albandechasteigner/GeniusLociForDynamo/assets/53272463/4309b3af-fb74-40a3-96e5-e62b699335d3)

**Comment by rvida5** [comment]
Hi Alban,

I currently have the latest version of the DynamoIronPython2.7 package installed (see image). It doesn't seem that is the problem.

![image](https://github.com/albandechasteigner/GeniusLociForDynamo/assets/53272463/59000fac-23db-4a05-bbb4-d7d8c9125c46)

**Comment by albandechasteigner** [comment]
Hi,

I confirm that the node works with Revit 2023. 
![image](https://github.com/albandechasteigner/GeniusLociForDynamo/assets/67761287/2a7a2a88-a9cd-4525-9fcf-54a63d15c75f)

I see that you have 1294 walls. Try with just a few hundred walls to see if it works.

**Comment by rvida5** [comment]
Thanks for the suggestion. I tried it on smaller and smaller lists of walls until I was able to narrow down the problem, it looks like there were some curtain walls included in the selection and that appears to cause an error with the node that prevents it from providing any results, even for the non curtain walls. Now that I know that's the reason, the solution is easy enough, I'll just update the graph to specifically filter out curtain walls... or filter out anything other than basic walls. Thanks!
